src/core_atmosphere/physics/physics_wrf/module_bl_ysu.F

```diff
@@ -1326,8 +1328,16 @@ subroutine ysu2d(j,ux,vx,tx,qx,p2d,p2di,pi2d,                               &
       enddo
  !Hybrid pblh of MYNN
  !tke is q2
-      CALL GET_PBLH(KTS,KTE,pblh_ysu(i),thvx(i,kts:kte),&
-      &    tke_ysu(i,kts:kte),zq(i,kts:kte+1),dzq(i,kts:kte),xland(i))
+!     CALL GET_PBLH(KTS,KTE,pblh_ysu(i),thvx(i,kts:kte),&
+!     &    tke_ysu(i,kts:kte),zq(i,kts:kte+1),dzq(i,kts:kte),xland(i))
+      do k = kts,kte
+         thvx_1d(k) = thvx(i,k)
+         tke_1d(k)  = tke_ysu(i,k)
+         zq_1d(k)   = zq(i,k)
+         dzq_1d(k)  = dzq(i,k)
+      enddo
+      zq_1d(kte+1) = zq(i,kte+1)
+      call get_pblh(kts,kte,pblh_ysu(i),thvx_1d,tke_1d,zq_1d,dzq_1d,xland(i))
```

Comment:
@ldfowler58 Why are these lines here in this PR?

Reply:
I forgot that I used an updated version of module_bl_ysu.F when I updated and tested the nTiedtke scheme. module_bl_ysu.F (see earlier PR) got inadvertently included in the main commit. Do not know how to fix this.

Reply:
I've just reverted the YSU PBL changed with a rebase.